src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -1896,20 +1896,25 @@ CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::_RequestSEDActiv
 
     if (!onOff && delayIdle && CHIP_DEVICE_CONFIG_SED_ACTIVE_THRESHOLD.count() != 0)
```

Comment:
I don't get why this change is needed. Shouldn't we rather fix this condition to be executed when `mActiveModeConsumers` drops to 0? Then everything should be OK because:
- if the timer expires while another exchange is active, then it will simply be a no-op becaue `mActiveModeConsumers` will be `> 0`
- if a new exchange is opened and close while the timer is scheduled, and the active consumers count drops to 0 again, then `DeviceLayer::SystemLayer().StartTimer` will cancel the previously scheduled timer (because that's how `StartTimer works if the same function and argument are provided).

On the other hand, we should schedule the timer when e.g. we decrease the number of active consumers from, say, 2 to 1.

Reply:
Oh, I read Boris comment. So the goal is to handle the case when someone first calls `RequestSEDActiveMode` with `delayIdle=true ` and then with `delayIdle=false`. That makes sense to me now. Disregard my comment :)